### ISSUE TITLE
[Bug]: "This plugin generated a warning from the characteristic"

### ISSUE THREAD

**Issue by amery78** [homebridge-eufy-security/plugin#116]
### What happened?

Hi all, first thank you for this fantastic plugin.
After having issues using this plugin on Hoops (not supported) I installed home bridge (Ver 1.5) and installed Eufy-security (last stable version).
Everything is running smoothly except this error message :

[7/18/2022, 11:29:49 PM] [homebridge-eufy-security] This plugin generated a warning from the characteristic 'Security System Target State': characteristic was supplied illegal value: number 63 exceeded maximum of 3. See https://homebridge.io/w/JtMGR for more info.

Of course I already went to the webpage and tried recommended actions and I also tried to find a solution on GitHub but with no success so far..

Could you help me ?

Thank you in advance,

Axel

### Device Type

other (UnSupported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

other (UnSupported)

### NodeJS Version

v16 (Supported)

### Operating System

Mac OS X 11+ (Supported)

### Relevant log output

```shell
[7/18/2022, 11:29:49 PM] [homebridge-eufy-security] This plugin generated a warning from the characteristic 'Security System Target State': characteristic was supplied illegal value: number 63 exceeded maximum of 3. See https://homebridge.io/w/JtMGR for more info.
```

**Comment by thieren** [comment]
I think this is already resolved in the latest beta. See #52 for reference.

**Comment by thieren** [comment]
As a workaround on current master you could set the default values for `hkHome`, `hkAway`, `hkNight` and `hkOff` in the config.

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
